plugins/woocommerce/src/Admin/API/Reports/Categories/Controller.php

```diff
@@ -235,13 +235,23 @@ public function get_collection_params() {
 			'description'       => __( 'Sort collection by object attribute.', 'woocommerce' ),
 			'type'              => 'string',
 			'default'           => 'category_id',
-			'enum'              => array(
-				'category_id',
-				'items_sold',
-				'net_revenue',
-				'orders_count',
-				'products_count',
-				'category',
+			/**
+			 * Filter to add or remove orderby params.
+			 *
+			 * @param array $orderby_enum Array of params permitted for orderby.
+			 *
+			 * @since 9.2.0
+			 */
+			'enum'              => apply_filters(
+				'woocommerce_report_categories_orderby_params',
```

Comment:
Most of the reports filters I see in the codebase are like

`woocommerce_reports_....` instead of `woocommerce_report_....` 

WDYT of keeping them consistent?